src/common/bilateralcl.h

```diff
@@ -35,7 +35,7 @@ typedef struct dt_bilateral_cl_t
 {
   dt_bilateral_cl_global_t *global;
   int devid;
-  size_t size_x, size_y, size_z;
+  int size_x, size_y, size_z;
```

Comment:
wondering if unsigned int won't be better here? Same question below.

Reply:
In theory, I guess yes. In practice, "all" width/height variables and struct members seem to be using `int`, so it would "look" weird and may lead to more surprising behavior than just having to keep one thing in your head. If it ever makes a difference anywhere...

Personally I'd rather be consistent than partially more "correct", but if you'd like me to change it, I won't object.